conductor_api/src/logger.rs

```diff
@@ -167,13 +170,6 @@ pub mod tests {
         assert_eq!(m.msg, "xboy");
     }
 
-    #[test]
-    fn test_log_rules_default() {
-        let rules = LogRules::default();
-        assert_eq!(rules.rules.len(), 3);
-        assert_eq!(format!("{:?}",rules),"LogRules { rules: [LogRule { pattern: ^err/, exclude: false, color: Some(\"red\") }, LogRule { pattern: ^debug/dna, exclude: false, color: Some(\"white\") }, LogRule { pattern: .*, exclude: false, color: None }] }".to_string());
-    }
-
```

Comment:
@zippy how do you feel about this test being removed? I feel this is too brittle and is not a useful test. It just caused my previous commit's build to break because I added another default rule.

Reply:
seems ok.